src/lib/components/footer.svelte

```diff
@@ -1,49 +1,36 @@
 <div class="container">
   <div>
-    <div>
-      <div class="social-media">
-        <h3>F&oslash;lg os</h3>
-        <a href="https://www.instagram.com">
-          <h3>Ig</h3>
-        </a>
-
-        <a href="https://es-es.facebook.com">
-          <h3>Fb</h3>
-        </a>
-      </div>
-      <h3>Ophavsret &#169; 2021,</h3>
-      <h3>Vermouth NU.</h3>
-      <h3>Drevet af Shopify</h3>
-    </div>
+    <p>Ophavsret &#169; 2023</p>
+    <p>Vermouth NU.</p>
+    <p>CVR - 40649395</p>
   </div>
-  <div>
-    <div class="right">
-      <h3>Newsletter</h3>
-      <h3>tlf:784-449-333</h3>
-      <h3>noseymeil@gmail.com</h3>
-      <h3>Nyhedsbrev</h3>
+  <div class="right">
+    <div class="social-media">
+      <p>F&oslash;lg os</p>
+      <a href="https://www.instagram.com"> Ig </a>
+      <a href="https://es-es.facebook.com"> Fb </a>
     </div>
+    <a href="mailto:info@vermouth.nu"> info@vermouth.nu </a>
   </div>
 </div>
 
 <style>
-  a h3 {
+  .social-media a {
     color: var(--interactive-button);
   }
   .social-media {
     gap: 10px;
     display: flex;
-  }
-
-  h3 {
-    font-size: var(--25px);
-    line-height: 1.5;
-    color: var(--secondary-text-color);
+    justify-content: flex-end;
```

Comment:
why is this here

Reply:
because if not social media aligns on the left

Reply:
justify-content ONLY affects flex containers, where is the flex coming from?